azure-pipelines/pull-request-validation/pr-msal.yml

```diff
@@ -0,0 +1,74 @@
+# File: azure-pipelines\pull-request-validation\pr-msal.yml
+# Description: Assemble and run unit test
+# Variable: 'ENV_VSTS_MVN_ANDROIDADACCOUNTS_USERNAME' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+# https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+name: $(date:yyyyMMdd)$(rev:.r)
+
+variables:
+- name: BuildParameters.jdkVersion
+  value: 1.8
+- name: BuildParameters.jdkArchitecture
+  value: x64
+- name: BuildParameters.javaHomeSelection
+  value: JDKVersion
+- name: robolectricSdkVersion
+  value: 28
+
+trigger:
+  branches:
+    include:
+    - dev
+    - master
```

Comment:
can we add release/*?

Reply:
done here 77ff9ab2fc8560f0006b5c7c0419d27935811452